.ci/opensearch/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+
+  opensearch:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - SECURE_INTEGRATION=${SECURE_INTEGRATION:-true}
+        - OPENSEARCH_VERSION=${OPENSEARCH_VERSION}
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+    ports:
+      - "9200:9200"
```

Comment:
We should also open up 9600 for [perf analyzer](https://github.com/opensearch-project/performance-analyzer)

Reply:
Currently clients don't support plugins, so the port added is only 9200 for OpenSearch.